package.json

```diff
@@ -76,19 +76,19 @@
     "linkifyjs": "^2.1.4",
     "lodash": "^4.17.2",
     "marked": "^0.3.6",
-    "material-ui": "^0.18.0",
+    "material-ui": "^0.19.4",
     "moment": "^2.17.1",
     "prop-types": "^15.5.10",
-    "react": "^15.6.1",
+    "react": "^16.1.0",
     "react-cookie": "^2.0.7",
-    "react-dom": "^15.6.1",
+    "react-dom": "^16.1.0",
     "react-draggable": "^3.0.3",
     "react-dropzone": "^4.1.3",
-    "react-highlight": "^0.10.0",
+    "react-highlight": "briancappello/react-highlight#react-v16-compiled",
```

Comment:
@ianwremmel what are your thoughts on having this in place until it either gets merged or we replace the code highlighting with a different repo?